crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -265,6 +265,55 @@ impl CairoHintProcessor<'_> {
                     .stop_mock_call(contract_address, &function_name);
                 Ok(())
             }
+            "start_spoof" => {
+                fn make_option<T>(is_value_set: bool, value: T) -> Option<T> {
+                    if is_value_set {
+                        Option::Some(value)
+                    } else {
+                        Option::None
+                    }
+                };
+
+                let contract_address = ContractAddress(PatriciaKey::try_from(StarkFelt::new(
+                    inputs[0].clone().to_be_bytes(),
+                )?)?);
+
+                let version = make_option(inputs[1].clone().is_one(), inputs[2].clone());
```

Comment:
What could be done here is just 

```rust
let version = inputs[1].is_one().then(|| inputs[2].clone());
```

No need to constructor a helper that basically achieves the same.